experimental/repositories/demologin_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import '../models/demouser.dart';
+
+class DemologinRepository {
+  static const String _password = "testtest";
+
+  static Future<DemoUser?> authentication(String email) async {
+    try {
+      UserCredential credential = await FirebaseAuth.instance.signInWithEmailAndPassword(email: email, password: _password);
```

Comment:
```suggestion
      UserCredential credential =
          await FirebaseAuth.instance.signInWithEmailAndPassword(
        email: email,
        password: _password,
      );
```
ちょっと長いので、コンマを入れて改行しましょう

Reply:
行を分割しました！